test/spec/modules/consentManagement_spec.js

```diff
@@ -45,6 +45,415 @@ describe('consentManagement', function () {
         expect(allowAuction).to.be.false;
       });
     });
+
+    describe('static consent string setConfig value', () => {
+      afterEach(() => {
+        config.resetConfig();
+        $$PREBID_GLOBAL$$.requestBids.removeHook(requestBidsHook);
+      });
+      it('results in user settings overriding system defaults', () => {
+        let staticConfig = {
+          cmpApi: 'static',
```

Comment:
Needs to be part of a `consentManagement` object:
```
pbjs.setConfig({
          consentManagement: {
            cmpApi: 'iab',
            timeout: 8000,
            allowAuctionWithoutConsent: false
          }
        });
```
This issue exists in another test in this file.

Reply:
This setup is working because the `setConfig` used in the unit test file is calling the local version of the function that's inside `consentManagement.js`; it's not being routed through the `config.setConfig()` function that requires config object names.  

From an off-line discussion, I plan to rewrite the unit tests to better reflect usage; ideally after this PR is finished.

Reply:
All right! Cool! Thanks for the clarification

Reply:
ok